### DISCUSSION TITLE
[Announcement] Black Arch Slim ISO release and breaking news on matters

### DISCUSSION THREAD

**Discussion by net-wayfarer** [BlackArch/blackarch#3045] [General]
Hi all,

Thank you for taking your time in reading this discussion first of all. The Black Arch team is happy to release a slimmed down variant of Black Arch in which we have meticulously worked to ensure that everything works as smooth as intended.

The issue with Calamares, the graphical installer, is that there is an issue with `dosfstools` which does not yet support version 4.2. This is current as of writing this disclosure in which Black Arch's upstream provider, [Arch Linux has this affected version in their repositories](https://github.com/archlinux/svntogit-packages/commit/f8c1a95e89726a264d2f801f0edd770369682618#diff-3e341d2d9c67be01819b25b25d5e53ea3cdf3a38d28846cda85a195eb9b7203a). As such, the Black Arch team has *temporarily* decided to downgrade `dosfstools` in order to workaround the lack of support. You can read more information about Calamares issue here:
* [kpmcore doesn't support dosfstools 4.2 yet, which fails to create GPT partitions #1637](https://github.com/calamares/calamares/issues/1637)

Please note that as stated above, **this is a temporary workaround** which only affects the installation phase, it does not affect the installed Black Arch after it has installed. Therefore, this disclosure is provided for transparency.

Affected version:
* BlackArch Linux 64 bit Slim ISO | 2021.03.01 | 465047fb89313167784e5655ea5a5c2fbaa50902

Fixed version:
* BlackArch Linux 64 bit Slim ISO | 2021.03.01 | 78a0c637b36af1ef0fbc48409b7982ee319f7ac5

The Black Arch team will continue to monitor and report any breaking news before and during the release. So please make sure to keep watch of this thread to ensure that you do not miss out!

**Comment by raj838-git** [comment]
If we install other blackarch tool category like nfc than not shown in blackarch menu.
Can you solve this problem ?

**Reply by Edu4rdSHL** [reply]
You need to install `blackarch-menus` then uninstall and install again the package.

**Reply by raj838-git** [reply]
Not proper working. If we install blackarch-reversing category tool android-apktool then not shown in reversing menu but apkstudio show in reversing menu.
Can you fix it.
